### PR TITLE
Add sensors support

### PR DESCRIPTION
**Pull request by vrunoa**
Prepare Android driver to support sensors set for Emulators implemented on ADB
https://github.com/appium/appium-adb/pull/456

### CODE REVIEW

File: lib/commands/actions.js [hunk 1]
```diff
@@ -312,6 +312,13 @@ commands.networkSpeed = async function networkSpeed (networkSpeed) {
   await this.adb.networkSpeed(networkSpeed);
 };
 
+commands.sensorSet = async function sensorSet (sensorType, value) {
```
Comment (KazuCocoa) [inline lib/commands/actions.js]: Can you add the docstring?

File: test/unit/commands/actions-specs.js [hunk 2]
```diff
@@ -352,6 +352,22 @@ describe('Actions', function () {
...
   });
+  describe('sensorSet', function () {
+    it('should call sensor adb command for emulator', async function () {
+      sandbox.stub(driver.adb, 'sensorSet');
+      sandbox.stub(driver, 'isEmulator').returns(true);
+      await driver.sensorSet('light', 0);
+      driver.adb.sensorSet.calledWithExactly('light', 0)
+        .should.be.true;
+    });
+    it('should throw exception for real device', async function () {
```
Comment (KazuCocoa) [inline test/unit/commands/actions-specs.js]: 👍

File: lib/commands/actions.js [hunk 3]
```diff
@@ -312,6 +312,20 @@ commands.networkSpeed = async function networkSpeed (networkSpeed) {
...
 };
 
+/**
+ * Emulate sensors values on the connected emulator.
+ *
+ * @param {string} sensorType - Sensor type declared in adb.SENSORS items.
+ * @param {number|string} value  - Number to set as the sensor value.
+ * @throws {Error} - If deviceType is not an emulator
+ */
+commands.sensorSet = async function sensorSet (sensorType, value) {
```
Comment (mykola-mokhnach) [inline lib/commands/actions.js]: which external API endpoint is supposed to trigger this method?

Comment (vrunoa) [reply lib/commands/actions.js]: @mykola-mokhnach this is something I want to discuss with the community. We have two options here; 
1. using one endpoint per type of sensor. For ex; W3C already has a method for light sensor in progress. https://www.w3.org/TR/ambient-light/
2. using one endpoint for all sensors. 
I'd like to hear you guys feedback. I'll probably start the conversation later today in slack

Comment (mykola-mokhnach) [reply lib/commands/actions.js]: I would say it is always safe to have a "mobile:" endpoint for such purpose

Comment (vrunoa) [reply lib/commands/actions.js]: Added a new mapping here: https://github.com/appium/appium-android-driver/pull/555/files#diff-2ac213fe13b12323b4767dd0bd037ee4R37

File: lib/commands/actions.js [hunk 4]
```diff
@@ -312,6 +312,28 @@ commands.networkSpeed = async function networkSpeed (networkSpeed) {
   await this.adb.networkSpeed(networkSpeed);
 };
 
+/**
+ * Emulate sensors values on the connected emulator.
+ *
+ * @param {object} opts - Object containing sensor type declared in adb.SENSORS items and value
```
Comment (mykola-mokhnach) [inline lib/commands/actions.js]: Please use appropriate format to describe this type. Check https://github.com/appium/appium-adb/blob/c00e48e9977f024ca3280e1ecf6b823997594779/lib/tools/system-calls.js#L141 as an example

**Comment by KazuCocoa** [comment]
We should require `^7.9.0` in https://github.com/appium/appium-android-driver/blob/master/package.json#L39 for sensor

**Comment by mykola-mokhnach** [comment]
LGTM. Do not forget to add the same mobile command to uia2 and espresso after this PR is published

**Pull request merged**